main.js

```diff
@@ -1,47 +1,78 @@
-// Math helpers
-function randomIntFromInterval(min, max) { // min and max included 
-    return Math.floor(Math.random() * (max - min + 1) + min)
-  }
-
+import {Circle} from './DomClasses/circle.js'
+import { scaleDownCircle,moveCircle } from './DomClasses/domInteractions.js'
 // Start button logic
-document.getElementById("startButton").addEventListener("click", function(){
-    let circlesElements = []
-    let bounceDiv = document.getElementById("bounceDiv");
-
-    let xCoord = Math.random()*100
-
-    let {top:topBounceDiv,left:leftBounceDiv,bottom:bottomBounceDiv,right:rightBounceDiv} = bounceDiv.getBoundingClientRect();
-
-    let diffRightLeft = rightBounceDiv-leftBounceDiv
-    let difTopBottom = bottomBounceDiv - topBounceDiv
 
-    const newDiv = document.createElement("div");
-    const circleRadius = 40
-    newDiv.style.width=circleRadius+"px"
-    newDiv.style.height=circleRadius+"px"
-    newDiv.classList.add("circle")
-    newDiv.style.left=(leftBounceDiv+randomIntFromInterval(0,diffRightLeft-circleRadius))+"px"
-    newDiv.style.top=(topBounceDiv+randomIntFromInterval(0,difTopBottom-circleRadius))+"px"
+const startWrapper = (function(){
 
-    console.log(newDiv.style.top)
-    bounceDiv.append(newDiv)
+    let isStartButtonEnable = true
+    let bounceInterval 
+    let circlesElements = []
+    let bounceDiv = document.getElementById("bounceDiv")
+    let ballCountInput = document.getElementById("ballCountInput")
 
-});
+    
+    return function(){
+        document.getElementById("startButton").addEventListener("click", function(){
+            if(isStartButtonEnable){
+                this.innerText = "Stop"
+                let ballsNumber = parseInt(ballCountInput.value)
+        
+                let {width:widthBounceDiv,height:heightBounceDiv,left:leftBounceDiv,top:topBounceDiv} = bounceDiv.getBoundingClientRect();
+                
+                for(let i=0;i<ballsNumber;i++){
+                    let newCircle = new Circle(widthBounceDiv,heightBounceDiv,leftBounceDiv,topBounceDiv)
+                    circlesElements.push(newCircle)
+                    bounceDiv.append(newCircle.dom)
+                }
+                
+        
+                bounceInterval = setInterval(()=>{
+                    let {top:topBounceDiv,left:leftBounceDiv,bottom:bottomBounceDiv,right:rightBounceDiv} = bounceDiv.getBoundingClientRect();
+                    circlesElements.forEach((circle)=>{
+                        moveCircle(circle,leftBounceDiv,rightBounceDiv,topBounceDiv,bottomBounceDiv)
+                    })
+                },16.6)
```

Comment:
Please extract this [magic number](https://stackoverflow.com/a/47902/612847) into a constant with a proper name.